Barotrauma/BarotraumaShared/SharedSource/StatusEffects/StatusEffect.cs

```diff
@@ -577,6 +577,8 @@ public AITrigger(XElement element)
 
         private readonly bool removeItem, dropContainedItems, dropItem, removeCharacter, breakLimb, hideLimb;
         private readonly float hideLimbTimer;
+        private readonly bool removeCharacterAndSaveItems;
+        private readonly string saveItemsContainerIdentifier;
```

Comment:
In my opinion it's not necessary to have two attributes for controlling this: we could just allow the user to define the container and determine whether to save the items based on whether it's defined or not.

The identifier should also be of the type `Identifier`, not a string.